app/src/main/res/values-xxhdpi/dimens.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <!-- Profile Chooser Fragment -->
+  <dimen name="profile_chooser_fragment_profile_select_text_margin_top">64dp</dimen>
+  <dimen name="profile_chooser_fragment_profile_recycler_view_margin_top">36dp</dimen>
+  <dimen name="profile_chooser_profile_view_view_margin_top">60dp</dimen>
+  <dimen name="profile_chooser_add_view_margin_top_profile_not_added">60dp</dimen>
+</resources>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
Done.